src/main.js

```diff
@@ -2,7 +2,7 @@ import App from './App.svelte';
 import './index.css';
 import { initIDAS } from './auth';
 
-const settings = await initIDAS(<INSERT YOUR AUTH TOKEN HERE>);
+const settings = await initIDAS('<INSERT YOUR AUTH TOKEN HERE>');
```

Comment:
I would leave the invalid code here so that an error is thrown - no quotes. The app can't function without the AppToken